windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -75,5 +105,3 @@ Windows nodes are now fully integrated with most of the monitoring capabilities
 * The filesystem graphs available in the web console do not display for Windows nodes. This is caused by changes in the filesystem queries. This will be fixed in a future release of WMCO. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930347[*BZ#1930347*])
 
 * The Prometheus windows_exporter used by the WMCO currently collects metrics through HTTP, so it is considered unsafe. You must ensure that only trusted users can retrieve metrics from the endpoint. The windows_exporter feature recently added support for HTTPS configuration, but this configuration has not been implemented for WMCO. Support for HTTPS configuration in the WMCO will be added in a future release.
```

Comment:
These issues still exists in 2.0.1

Reply:
Moved this section to a higher level.

Reply:
I will defer to the docs folks on this as I don't know if we want to track version history.